projects/angular/src/forms/common/label.ts

```diff
@@ -18,6 +30,7 @@ import { NgControlService } from './providers/ng-control.service';
 export class ClrLabel implements OnInit, OnDestroy {
   @Input('id') idInput: string;
   @HostBinding('attr.id') idAttr: string;
+  @ContentChild(ClrSignpost, { read: ElementRef }) signpost: ElementRef;
```

Comment:
```suggestion
  @ContentChild(ClrSignpost, { read: ElementRef }) private signpost: ElementRef;
```

This should be `private` as well.